### PR TITLE
add the ci build

### PR DESCRIPTION
**Pull request by wenchajun**
This PR will configure the ability to automatically package images to dockerhub, but first, you need to configure  `secrets.REGISTRY_USER` and `secrets.REGISTRY_PASSWORD`    fields.
fix https://github.com/kubesphere/kube-events/issues/30

Signed-off-by: chengdehao <wenchajun@gmail.com>

### CODE REVIEW

File: .github/workflows/build-exporter-image.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+on:
+  push:
+    branches:
+      - 'master'
+    tags:
+      - 'v*'
+    paths:
+      - ".github/workflows/build-exporter-image.yaml"
+      - "cmd/operator/**"
```
Comment (junotx) [inline .github/workflows/build-exporter-image.yaml:11]: Here may be `cmd/exporter/**`

File: .github/workflows/build-ruler-image.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+on:
+  push:
+    branches:
+      - 'master'
+    tags:
+      - 'v*'
+    paths:
+      - ".github/workflows/build-ruler-image.yaml"
+      - "cmd/operator/**"
```
Comment (junotx) [inline .github/workflows/build-ruler-image.yaml:11]: Here may be `cmd/ruler/**`

**Review by junotx — CHANGES_REQUESTED** [review]

File: .github/workflows/build-exporter-image.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+          password: ${{ secrets.REGISTRY_PASSWORD }}
+
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Build and Push Image for kube-events-exporter
+        run: |
+          docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+          make cross-build-exporter -e REPO_EXPORTER=${{ env.REPO_EXPORTER }}/kube-events-exporter
```
Comment (zhu733756) [inline .github/workflows/build-exporter-image.yaml:53]: ` docker run --rm --privileged multiarch/qemu-user-static --reset -p yes` would be useful for custom building, I think it integrating it in the Makefile is suitable.

Comment (zhu733756) [reply .github/workflows/build-exporter-image.yaml:53]: > or use this action [docker/setup-qemu-action](https://github.com/docker/setup-qemu-action) instead

Yeah. I am confusing about that some image like fluentd perhaps not supported this action, refer to https://github.com/fluent/fluentd-docker-image/blob/master/v0.14/debian-onbuild/hooks/post_push.  Maybe we need a test for it. For common speaking, this action would support.

**Review by benjaminhuo — APPROVED** [review]

**Pull request merged**